thumbor/Dockerfile

```diff
@@ -19,20 +19,21 @@ ENV SHELL bash
 ENV WORKON_HOME /usr/src/app
 WORKDIR /usr/src/app
 
-ARG DOCKERHOST=172.17.0.1
-ENV DOCKERHOST ${DOCKERHOST}
 COPY requirements.txt /usr/src/app/requirements.txt
-RUN pip install --trusted-host None --no-cache-dir --use-wheel \
-   --extra-index-url http://${DOCKERHOST}:9009/simple/ \
-   --trusted-host ${DOCKERHOST} \
+RUN pip install -U pip==9.0.3
+RUN pip install --trusted-host None --no-cache-dir \
    -r /usr/src/app/requirements.txt
 
 COPY conf/thumbor.conf.tpl /usr/src/app/thumbor.conf.tpl
 
-RUN \ 
+RUN \
     ln /usr/lib/python2.7/dist-packages/cv2.x86_64-linux-gnu.so /usr/local/lib/python2.7/cv2.so && \
```

Comment:
are you sure this is still required?

Reply:
I wasn't really sure what it's for, so just left it... Any idea how to test whether it's still required or not?

Reply:
if starting thumbor does not produce warning in console, then it's good. 
It is probably still needed, for opencv-engine.

Reply:
Looks like it. I tried to remove it, and got an error

`ImportError: No module named cv2`

I think I'll leave it in then.